src/main/java/racingcar/service/CarRacing.java

```diff
@@ -0,0 +1,52 @@
+package racingcar.service;
+
+import racingcar.model.Car;
+import racingcar.model.CarRecord;
+
+import java.util.*;
+
+public class CarRacing {
+
+    private List<Car> cars;
+
+    public CarRacing() {}
+
+    public CarRacing(int numberOfCars) {
+        carRaceReady(numberOfCars);
+    }
+
+    private void carRaceReady(int numberOfCars) {
+        cars = new ArrayList<Car>();
+
+        for(int i = 0; i < numberOfCars; i++) {
+            cars.add(new Car());
+        }
+    }
+
+    public List<CarRecord> carRaceStart() {
+        List<CarRecord> records = new ArrayList<>();
+
+        for(Car car : cars) {
+            car.move(isCarMovingForward());
+            CarRecord record = new CarRecord(car.getMovingDistance());
+            records.add(record);
+        }
+
+        return records;
+    }
+
+    public boolean isCarMovingForward() {
+        Random random = new Random();
+        int randomNumber = random.nextInt(9) + 1;
+
+            if(randomNumber >= 4)
+                return true;
+
+        return false;
```

Comment:
```suggestion
        Random random = new Random();
        int randomNumber = random.nextInt(9) + 1;
        return randomNumber >= 4;
```
이렇게 줄여볼 수 있겠네요

추가로 4가 무엇인지 의미 부여해보면 좋을 것 같습니다.